src/main.rs

```diff
@@ -97,6 +109,8 @@ pub async fn main() -> Result<(), anyhow::Error> {
 
     log::info!("=> Starting server on {}", addr.to_string());
 
+    let builder = Router::builder();
```

Comment:
I must be misunderstanding something, but how come this is defined here, but only used once on line 154 where it gets masked by another `let builder`?

Reply:
Because I took a break in between writing it 😆 . I'll fix that right now